draft-ietf-webtrans-http3.md

```diff
@@ -456,22 +472,19 @@ beginning of the stream; resetting a stream might result in that data being
 discarded when using a RESET_STREAM frame.  To prevent this, WebTransport
 implementations MUST use the RESET_STREAM_AT frame {{RESET-STREAM-AT}} with a
 Reliable Size set to at least the size of the WebTransport header when
-resetting a WebTransport data stream.  This ensures that the ID field
-associating the data stream with a WebTransport session is always delivered.
+resetting a WebTransport data stream.  This ensures reliable delivery of the ID
+field associating the data stream with a WebTransport session.
 
-WebTransport implementations SHALL forward the error code for a stream
-associated with a known session to the application that owns that session;
-similarly, the intermediaries SHALL reset the streams with corresponding error
-code when receiving a reset from the peer.  If a WebTransport implementation
-intentionally allows only one session over a given HTTP/3 connection, it SHALL
-forward the error codes within WebTransport application error code range to the
-application that owns the only session on that connection.
```

Comment:
This removal is a bit outside the scope of the PR, but I think that it's a good change.

Reply:
Was this referring to connection close application error codes?

Reply:
Yeah, I ended up removing the second half here because it seemed duplicative with the part above. Happy to break that out as needed, but really what we're saying is: 
- If you have a WebTransport stream for which you know the session and it gets a WebTransport error, send that to the application
- If you have only one session on the H3 connection, you always know the session

That second part didn't seem that useful to state, since in theory we've already made it so that you always know the session on a stream (see reset-stream-at)